rust/plan.md

```diff
@@ -1,27 +1,14 @@
 # Practical Networked Applications in Rust - Lesson plan
 
 This course is divided into five _sections_, each with a _project_ designed to
-give you exposure to practical Rust programming subjects. Each section's
-_lessons_ and accompanying external pre-requisite _readings_ provide the
-resources necessary to complete the projects.
-
-Note that it is not sufficient to follow the lessons without reading the
-external reading material. The readings are the foundation necessary to
-understand and complete the projects; the lessons themselves are intended to
-provide additional insight and in-depth explorations beyond the background
-reading.
+give you exposure to practical Rust programming subjects.
 
 Each project builds on experience from the previous project. It is reasonable to
 start each project by copy-pasting the previous.
 
 A suggested workflow:
```

Comment:
I find most external readings are closely related to the lessons but not so relevant to the project itself. So I remove external readings as well.

Then, we have two things reduced in the workflow: reading the external materials and following the lessons. I doubt if it can be called a **workflow** now.

Reply:
I do think we need to have readings because the projects themselves don't offer any instruction at all - they are to exercise what the student has learned. You are right though that many if not all of the listed readings are oriented towards the lessons and not the projects. Anyway they are fine to remove. We can reconsider them later.

I agree that describing a workflow isn't necessary anymore. Will you adjust the patch to remove it entirely? Then you can merge the PR yourself.

Reply:
Actually GitHub tells me "The base branch restricts merging to authorized users." instead of CI failures.